.travis.yml

```diff
@@ -6,12 +6,12 @@ os: osx
 env:
   global:
     - CACHE="${HOME}/.local"
-    - MPICH_VER="3.2"
+    - MPICH_VER="3.2.1"
```

Comment:
Homebrew defaults to the latest stable version of a package unless you "pin" it. Since I try to reuse info across the build matrix, telling everyone (who needs to be explicitly told) to use the same version as Homebrew simplifies CI infra maintenance. Also, I was hoping that a patch Andre submitted to MPICH might be in the 3.2.1 release but that doesn't appear to be the case.